src/site/content/en/blog/aspect-ratio/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Maintaining Aspect Ratio in CSS'
+subhead: The new aspect-ratio property can help with responsive dimension-based imaged.
+authors:
+  - una
+date: 2021-01-14
+hero: hero.jpg
+alt: Image of colorful telephones resized with the same aspect ratio.
+description: There are several ways to maintain aspect ratio and image dimensions on the web. This post goes over two ways—one popular hack, and one new CSS property.
+tags:
+  - blog
+  - css
+---
+
+When resizing some media, in many cases, a design calls to maintain this ratio
+and ensure the image resizes in an even distribution. This width-to-height ratio
+is called the *aspect-ratio*.
+
+Aspect ratio is most commonly expressed as two integers and a colon in the
+dimensions of: width:height, or x:y. The most common aspect ratios for
+photography are 4:3 and 3:2, while video, and more recent consumer cameras, tend
+to have a 16:9 aspect ratio.
+
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aspect-ratio.jpg" alt="">
+  <figcaption class="w-figcaption">
+    Two images with the same aspect ratio. One is 634 x 951px while the other is 200 x 300px. Both have a 2:3 aspect ratio.
+  </figcaption>
+</figure>
+
+## Why maintain aspect ratio
+
+With the advent of responsive design, maintaining aspect ratio has been
+increasingly important for web developers, especially as image dimensions
+differ. At times, it is difficult to maintain consistent layout sizing and place
+media like [video](http://fitvidsjs.com/) and background images within a parent
+block.
```

Comment:
This doesn't explain why you should maintain aspect ratio. My naive understanding is that you want to maintain it because if you don't your images will look distorted/unprofessional/weird